README.md

```diff
@@ -108,16 +106,23 @@ Example command:
 
 ### Setup
 Clone the repository to desired directory on local computer.
-> `git clone https://github.com/wacoo/vet-clinic-database`
+> `git clone https://github.com/wacoo/clinic-database`
 
-> `cd vet-clinic-database`
-> `run the SQL statements in the schema.sql, data.sql and queries.sql`
+> `cd clinic-database`
+> `run the SQL statements in the schema_based_on_diagram.sql`
 
 ## 👥 Author <a name="authors"></a>
+
 - GitHub: [@wacoo](https://github.com/wacoo)
 - Twitter: [@wabaham9](https://twitter.com/wabaham9)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/wondmagegn-abriham-b867289a)
 
+👤 **Faisal Mumtaz**
+
+- GitHub: [@faisalakb](https://github.com/faisalakb)
+- Twitter: [@faisalakb786](https://twitter.com/Faisalakb786)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/faisal-mumtaz-514a221a6/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly check your `License` link that is pointing to a `error 404` page and it is not good for user experience at the moment. When I look further, I found that the file you were referencing was not in your `dir`. Please fix this :point_down: 
![image](https://github.com/wacoo/clinic-database/assets/99715640/2969c2d1-2ccf-46ba-9924-d18e9151e7f1)